src/display/api.js

```diff
@@ -132,6 +133,8 @@ if (typeof PDFJSDev !== 'undefined' &&
  * @property {string} docBaseUrl - (optional) The base URL of the document,
  *   used when attempting to recover valid absolute URLs for annotations, and
  *   outline items, that (incorrectly) only specify relative URLs.
+ * @property {Object} canvasFactory - (optional) The factory that will be used
```

Comment:
Move `canvasFactory` from here and add to the RenderParameters below. We don't need to store canvas factory on the transport but rather than on a rendering task.